pandas/core/indexes/base.py

```diff
@@ -3556,7 +3556,7 @@ def get_slice_bound(self, label, side, kind):
 
     def slice_locs(self, start=None, end=None, step=None, kind=None):
         """
-        Compute slice locations for input labels.
+        For an ordered index, compute slice locations for input labels.
```

Comment:
Index does not need to be ordered (it's only when there are duplicates that it has to be ordered I think). eg:

```
In [11]: pd.Index([1, 3, 4, 2]).slice_locs(1, 2)
Out[11]: (0, 4)
```

Reply:
Oh, thanks for catching that. Iøve reworded.